client/deposits/strings.ts

```diff
@@ -111,7 +111,7 @@ export const payoutFailureMessages: Record< PayoutFailureCode, string > = {
 		'woocommerce-payments'
 	),
 	issuer_unavailable: __(
-		'The issuing bank is currently unavailable. Please wait a bit before trying again, or switch to a different payment method or card.',
+		'The issuing bank is currently unavailable. Our system will automatically try again on your next payout date, or you can switch to a different payout method.',
```

Comment:
8. `issuer_unavailable` – Provide more accurate information about the automatic retry process and available options.